x-pack/plugins/cases/common/api/cases/comment/index.test.ts

```diff
@@ -843,6 +843,28 @@ describe('Comments', () => {
         right: defaultRequest,
       });
     });
+
+    describe('errors', () => {
+      it(`throws error when attachments are more than ${MAX_BULK_CREATE_ATTACHMENTS}`, () => {
+        const comment = {
+          comment: 'Solve this fast!',
+          type: CommentType.user,
+          owner: 'cases',
+          foo: 'bar',
```

Comment:
Could you remove the `foo: 'bar'` field that is related to a different error? (I did the same mistake 😄 )